prep-css-box-model/style.css

```diff
@@ -0,0 +1,15 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-style: double;
+  border-color: darkgray;
+  border-width: 3px
```

Comment:
These 3 rules can be combined into a single CSS rule:

```CSS
border: 3px double darkgray;
```

Reply:
yep, that's useful. I saw that in the next exercise after I had submitted this one!